src/Uno.UWP/Globalization/NumberFormatting/NumeralSystemTranslator.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Uno.Globalization.NumberFormatting;
+
+namespace Windows.Globalization.NumberFormatting
+{
+	public partial class NumeralSystemTranslator
+	{
+		private readonly static Dictionary<string, char[]> _numeralSystemToDigitsDictionary =
```

Comment:
@jeromelaban Should I use switch/case instead of dictionary here?

Reply:
Yes, let's do that as well. Thanks!